datasets/gem/gem.py

```diff
@@ -1212,19 +1212,19 @@ def _generate_examples(self, filepath, split, filepaths=None, lang=None):
         elif self.config.name == "wiki_auto_asset_turk":
             if split in ["train", "validation"]:
                 keys = [
-                    "target_id",
-                    "source_id",
-                    "target",
                     "source",
+                    "target",
                 ]
                 with open(filepath, encoding="utf-8") as f:
                     for id_, line in enumerate(f):
                         values = line.strip().split("\t")
-                        assert len(values) == 5, f"Not enough fields in ---- {line} --- {values}"
-                        example = dict([(k, val) for k, val in zip(keys, values[1:])])
+                        assert len(values) == 2, f"Not enough fields in ---- {line} --- {values}"
+                        example = dict([(k, val) for k, val in zip(keys, values)])
                         example["gem_id"] = f"{self.config.name}-{split}-{id_}"
                         example["gem_parent_id"] = example["gem_id"]
                         example["references"] = [] if split == "train" else [example["target"]]
+                        example["source_id"] = ""
+                        example["target_id"] = ""
```

Comment:
Here instead of setting them to an empty string we can just remove them from the `features` defined here ?

https://github.com/huggingface/nlp/blob/c906e854111f11f576c477a230bfd41845c722b1/datasets/gem/gem.py#L486-L487